src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +249,57 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+
+    .pf-c-data-list__item-draggable-button {
```

Comment:
does this need to be nested in `.pf-m-draggable`? Doesn't seem like it depends on that parent class for anything.

Reply:
yeah i was meaning to ask you about this. i agree it doesnt have to be inside of `pf-m-draggable`